purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1336,30 +1336,32 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             @BillingClient.FeatureType feature: String, context: Context,
             callback: Callback<Boolean>
         ) {
-            BillingClient.newBuilder(context).setListener { _, _ -> }.build().let { billingClient ->
-                billingClient.startConnection(
-                    object : BillingClientStateListener {
-                        override fun onBillingSetupFinished(billingResult: BillingResult) {
-                            try {
-                                billingClient.endConnection()
-                                val featureSupported = billingClient.isFeatureSupported(feature)
-                                callback.onReceived(featureSupported.responseCode == BillingClient.BillingResponseCode.OK)
-                            } catch (e: IllegalArgumentException) {
-                                // Play Services not available
-                                callback.onReceived(false)
+            BillingClient.newBuilder(context).enablePendingPurchases().setListener { _, _ -> }
+                .build()
```

Comment:
nitpick: 
I think it'd be easier to read these and not miss an operation by doing one per-line, i.e.:
```kotlin
BillingClient.newBuilder(context)
    .enablePendingPurchases()
    .setListener { _, _ -> }
    .build()
    .let {
```